[TASK] Update to PHPUnit 6

oliverklee opened the pull request:
Note: Before this PR can be merged, #44 and #45 must be merged, and I will need to rebase this PR.

michield commented:
Hmm, I merged in the wrong order and now this one fails. Fixing it in Github is very clumsy.

oliverklee commented:
I usually fix this locally, not on GitHub. :-)

I've just rebased (and fixed the conflicts) and re-pushed.

@michield After merging, could you please directly delete the branch? This will help keep things clean. Thanks! <3
